src/credentials/dto/credentialOptions.dto.ts

```diff
@@ -45,14 +54,15 @@ export class CredentialOptionsDTO {
   @IsString()
   domain?: string
 
-  @ApiPropertyOptional({ type: () => CredentialStatusDTO })
-  @IsOptional()
+  @ApiProperty({ type: () => CredentialStatusDTO })
+  @ValidateIf((object, value) => value !== undefined)
```

Comment:
same as above

theoritically, `@IsOptional()` vs `@ValidateIf( !undefined )` should do the same thing.

why the change?
is `@IsOptional` not working?
should we have a ticket to place it everywhere?

Reply:
The difference is with regards to how we handle `null` values, this way, nulls are rejected.